images/docs-builder/links_ignorelist.json

```diff
@@ -2,6 +2,7 @@
     "ignorePatterns": [
         { "pattern": "^https://api.crc.testing" },
         { "pattern": "^http://api.crc.testing" },
+        { "pattern": "^https://docs.github.com/.*$" },
```

Comment:
@cfergeau following patch doesn't solve this one.

```patch
diff --git a/images/docs-builder/links_ignorelist.json b/images/docs-builder/links_ignorelist.json
index ad6348ef..bcd3e13d 100644
--- a/images/docs-builder/links_ignorelist.json
+++ b/images/docs-builder/links_ignorelist.json
@@ -2,9 +2,15 @@
     "ignorePatterns": [
         { "pattern": "^https://api.crc.testing" },
         { "pattern": "^http://api.crc.testing" },
-        { "pattern": "^https://docs.github.com/.*$" },
         { "pattern": "https://console-openshift-console.apps-crc.testing" },
         { "pattern": "http://proxy.example.com" }
+    ],
+    "httpHeaders": [
+        {
+          "urls": ["https://docs.github.com"],
+          "headers": {
+            "Accept-Encoding": "gzip, deflate"
+          }
+        }
     ]
-
 }
```

I am also confused why it is not working because with curl it does work
```
$ curl -H "Accept-Encoding: gzip, deflate" -I https://docs.github.com  
HTTP/2 302 
cache-control: private, no-store
content-type: text/plain; charset=utf-8
location: /en
```

FYI I did updated the doc-builder image after this change to make sure it is part of that build image.

Reply:
https://github.com/code-ready/crc/pull/3155 works locally for me.

Reply:
@cfergeau #3155 PR created against master branch instead of main branch.